plugin/src/test/resources/modules/org/wildfly/plugins/bootablejar/maven/goals/main/module.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module xmlns="urn:jboss:module:1.8" name="org.wildfly.plugins.bootablejar.maven.goals">
```

Comment:
@jamezp , why do we need 2 identical module.xml files in 2 different locations?

Reply:
Oops we don't. I added the when I was testing various ways of doing this. I'll delete the duplicate. Nice catch!